tequilapi/http_routes.go

```diff
@@ -2,16 +2,21 @@ package tequilapi
 
 import (
 	"github.com/julienschmidt/httprouter"
+	"github.com/mysterium/node/ipify"
 	"github.com/mysterium/node/tequilapi/endpoints"
 	"os"
 	"time"
 )
 
+// NewAPIRouter returns new api router with status endpoints
 func NewAPIRouter() *httprouter.Router {
 	router := httprouter.New()
 	router.HandleMethodNotAllowed = true
 
 	router.GET("/healthcheck", endpoints.HealthCheckEndpointFactory(time.Now, os.Getpid).HealthCheck)
 
+	ipifyClient := ipify.NewClient()
```

Comment:
Dont factory on the fly, pass dependency from upper layer

Reply:
But I just noticed that `NewAPIRouter()` factory is not right place to connect features

Reply:
Moved to connection endpoints

Reply:
Also, added dependency injection.